<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryWindow</class>
 <widget class="QMainWindow" name="EntryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>480</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="entryTitle" stdset="0">
   <string>EntryWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_show_pass">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать пароль</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_entry">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Далее</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_login">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pass">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_log">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="label_rem_pass">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: blue</string>
    </property>
    <property name="text">
     <string>Забыли пароль?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickableLabel" name="label_reg_acc">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
